patches/0013-os-support-UNC-paths-and-.-segments-in-fixLongPath.patch

```diff
@@ -0,0 +1,211 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: qmuntal <qmuntaldiaz@microsoft.com>
+Date: Tue, 12 Mar 2024 12:26:53 +0100
+Subject: [PATCH] os: support UNC paths and .. segments in fixLongPath
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
```

Comment:
Interesting extra headers--caused by `µ ±` etc. in the description. It's not a problem and round-trips properly, just thought I'd mention it in case of curiosity.